src/graph/network.cc

```diff
@@ -169,9 +169,9 @@ DGL_REGISTER_GLOBAL("network._CAPI_ReceiverRecvSubgraph")
                                           &(nf->layer_offsets),
                                           &(nf->flow_offsets));
       nf->graph = GraphPtr(new ImmutableGraph(csr, nullptr));
-      std::vector<NodeFlow*> subgs(1);
-      subgs[0] = nf;
-      *rv = WrapVectorReturn(subgs);
+      List<NodeFlow> subgs;
+      subgs.push_back(nf);
+      *rv = subgs;
```

Comment:
If we have only one nodeflow here, can we still use `*rv = &nf` here?

Reply:
Just use `*rv=nf`. Nodeflow has inherited `ObjectRef` so it is now a reference object (equivalent to a shared pointer).